src/generators/node_5_es6/index.js

```diff
@@ -101,16 +103,16 @@ function getEndpointUriStr(operation) {
 }
 
 function getQueryParameterNames(operation) {
-  return getQueryParameters(operation).map(param => ({
+  return map(getQueryParameters(operation), param => ({
     name: param.name,
     nameCamelCase: camelCase(param.name),
   }));
 }
 
 function getDeclaredResponses(operation) {
-  return operation.responses
-    .filter(response => response.code && response.code.integer)
-    .map((response) => {
+  return map(
```

Comment:
I saw a few errors in the logs where resources was `undefined`, introducing lodash to help prevent fatal errors.